python/fastquant/backtest/backtest.py

```diff
@@ -405,14 +411,19 @@ class CustomData(bt.feeds.PandasData):
                 **optim_params
             )
     # drop extra columns #248
-    if len(set(['channel' 'symbol']).intersection(sorted_combined_df.columns.values)) == 2:
-        sorted_combined_df.drop(['channel', 'symbol'], axis=1, inplace=True)
+    if (
+        len(
+            set(["channel" "symbol"]).intersection(
+                sorted_combined_df.columns.values
+            )
+        )
+        == 2
+    ):
+        sorted_combined_df.drop(["channel", "symbol"], axis=1, inplace=True)
```

Comment:
we might need to extend the character length in pyproject.toml for black formatter longer than the default

Reply:
@jpdeleon i don't know this.

Reply:
Ah it is just the formatting option here: https://github.com/enzoampil/fastquant/blob/master/python/pyproject.toml
The default is `line-length = 79` and it's probably better to make this a bit longer so the formatting won't cut the lines above awkwardly. Not crucial though, just aesthetics.

Reply:
ah I see. thanks for that :D